src/js/Views/RaceTrackView/components/createSpinElement.js

```diff
@@ -0,0 +1,13 @@
+function createSpinElement() {
+  const spinElementWrapper = document.createElement('div');
+  spinElementWrapper.classList.add('d-flex', 'justify-center', 'mt-3');
+  spinElementWrapper.innerHTML = `
+    <div class="relative spinner-container">
+      <span class="material spinner"></span>
+    </div>
+  `;
+
+  return spinElementWrapper;
+}
+
+export { createSpinElement };
```

Comment:
```suggestion
export function createSpinElement() {
  return createElement(`
      <div class="d-flex justify-center mt-3">
        <div class="relative spinner-container">
          <span class="material spinner"></span>
        </div>        
      </div>
  `);
}
```
이렇게 사용할 수 있도록 `createElement` 라는 함수를 정의해서 사용해본다면 어떨까요?